pkg/xen-tools/patches-4.19.0-rc4-v3/x86_64/05-xen-spoofing.patch

```diff
@@ -0,0 +1,217 @@
+diff --git a/tools/firmware/hvmloader/hvmloader.c b/tools/firmware/hvmloader/hvmloader.c
+index 598a226..0c3963a 100644
+--- a/tools/firmware/hvmloader/hvmloader.c
++++ b/tools/firmware/hvmloader/hvmloader.c
+@@ -135,9 +135,11 @@ static void init_hypercalls(void)
+ 
+         if ( !strcmp("XenVMMXenVMM", signature) )
+             break;
++        if ( !strcmp("ZenZenZenZen", signature) )
++            break;
+     }
+ 
+-    BUG_ON(strcmp("XenVMMXenVMM", signature) || ((eax - base) < 2));
```

Comment:
The new signature is added and checked, so I don't think we should remove this BUG_ON.... also, please, do not comment code, if the code is not needed, just remove it....